docs/EXAMPLES.md

```diff
@@ -244,6 +244,30 @@ describe("foo", () => {
 });
 ```
 
+> Note: this function will be deprecated in favor of the function `click`.
```

Comment:
This 'Note' looks better below the method title that will be depreciated.

Reply:
When debugging I comment the headless config and this turns to be useful.

Reply:
Good point. I'll update.